SCAutolib/models/file.py

```diff
@@ -298,6 +297,14 @@ def __init__(self):
             return
         self.__initialized = True
 
+        with open('/etc/redhat-release', "r") as f:
+            release = f.read()
+
+        if "release 8" in release or "release 9" in release:
```

Comment:
Should we reuse here the os version from utils?

Reply:
I tried but we have some circular deps that prevent as from that. Either way, it will be deprecated by the distro package.